### ISSUE TITLE
rendering seem borked after 0.5.5 -> 0.6 upgrade

### ISSUE THREAD

**Issue by sanga** [jupyterlab/jupyterlab_launcher#28]
I just upgraded jupyterlab_launcher from 0.5.5 to 0.6 this morning. And after restarting jupyter-lab I was greeted with the following:

<img width="775" alt="screen shot 2017-12-04 at 9 55 14" src="https://user-images.githubusercontent.com/2088/33541930-ae634f8a-d8d9-11e7-86d4-fdac0126efc7.png">

After downgrading to 0.5.5 things looked normal again:

<img width="921" alt="screen shot 2017-12-04 at 9 55 34" src="https://user-images.githubusercontent.com/2088/33541934-b28525ac-d8d9-11e7-9959-1ff957e9d72c.png">

The upgrade was done via pip3 install -U jupyterlab_launcher. Is there something else that needs to be done or?

For reference:

```
pip3 list | grep jupyter
jupyter-client (5.1.0)
jupyter-core (4.4.0)
jupyterlab (0.29.2)
jupyterlab-launcher (0.5.5)
```

**Comment by blink1073** [comment]
Hi @sanga, we released 0.6 of this ahead of JupyterLab 0.30 (should be released today).  0.6 of this library is not compatible with any currently released versions of JupyterLab.

**Comment by sanga** [comment]
Fair enough
